tests/runtests.py

```diff
@@ -321,6 +321,8 @@ class ActionSelenium(argparse.Action):
     Validate the comma-separated list of requested browsers.
     """
     def __call__(self, parser, namespace, values, option_string=None):
+        # Check if selenium is installed.
+        import selenium  # noqa
```

Comment:
I'd raise ``ImproperlyConfigured`` as in other cases:
```suggestion
        try:
            import selenium
        except ImportError as e:
            raise ImproperlyConfigured(f'Error loading selenium module: {e}')
```